EmptyBlock/Example3/config.xml

```diff
@@ -23,7 +23,7 @@
         <property name="javaParseExceptionSeverity" value="ignore"/>
 
         <module name="EmptyBlock">
-            <property name="tokens" value="LITERAL_DEFAULT"/>
+            <property name="tokens" value="LITERAL_CASE,LITERAL_DEFAULT"/>
```

Comment:
https://checkstyle.org/checks/blocks/emptyblock.html#Example3-config

It is not what we have.
https://github.com/checkstyle/checkstyle/tree/master/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock there are only 2 examples 

3rd at https://github.com/checkstyle/checkstyle/blob/master/src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/Example3.java
It was recently uploaded

Reply:
All good here, no action is required.